p/api/pshb.php

```diff
@@ -128,8 +128,8 @@
 	try {
 		Minz_Session::_param('currentUser', $username);
 		Minz_Configuration::register('user',
-		                             join_path(USERS_PATH, $username, 'config.php'),
-		                             join_path(FRESHRSS_PATH, 'config-user.default.php'));
+									 join_path(USERS_PATH, $username, 'config.php'),
+									 join_path(FRESHRSS_PATH, 'config-user.default.php'));
```

Comment:
Is the extra whitespace on purpose?

Reply:
Nope, phpcbf made the correction automatically. It's probably because we have some kind of weird rules for checking the syntax.
Maybe we should move to defacto standards like PSR